app/jobs/subscription_placement_job.rb

```diff
@@ -50,6 +50,7 @@ def place_order(order)
     return handle_empty_order(order, changes) if order.line_items.where('quantity > 0').empty?
 
     move_to_completion(order)
+    order.process_payments! if order.payment_required?
```

Comment:
:warning: this will likely conflict with https://github.com/openfoodfoundation/openfoodnetwork/pull/7281

Reply:
Yeah, it definitely will :) I think we can just add the same line right after we call `move_to_completion` again and it should be fine.

Reply:
Rebased to address this 👍